Dockerfile-elm

```diff
@@ -1,13 +1,10 @@
 FROM node:6.9.1
+RUN npm install -g yarn gulp
```

Comment:
Gulp is installed in line 8 (`yarn install`) already, as it is listes in `packages.json`.